### ISSUE TITLE
Accessibility updates to collection_set_detail and collection_detail

### ISSUE THREAD

**Issue by lwrubel** [gwu-libraries/sfm-ui#1024]
Collection_set_detail:
- "i" button fix: recommend using <button> if possible. Use <button aria-label="Info"> around the \<i class="fa ..." aria-hidden="true"> . If not possible, Max recommends <a role="button"> + JS. OR consider something simpler, such as removing the modal altogether and just provide a link to the documentation, with the link text being "Details about each collection type".

Collection_detail:
- Add alt-text to platform image (e.g. twitter bird) at top of page
- Add alt-text to twitter icon in seed list, including that it goes to the twitter page for that account.
- Add visible label to seedlist searchbox. <label for="search-input">Search</label><input id="search-input" ...>
- On recommended using <button> if possible (<a role="button"> + JS if not), but I had skimmed through the FA page too quickly when writing the original comment recommending the sr-only approach. I think making a <button aria-label="Info"> around the \<i class="fa ..." aria-hidden="true"> should be good.
- In harvests list, make the Date Requested data a link instead of the Type. 
- Headings not nested properly. Style instead. 
- Change log is using an id attributes that aren't unique. From SiteImprove: "Element ID is not unique":  id=div_id_history_note is repeated. 
- Change log heading not nested properly, make it an h2 not h4

Seed_detail page:
- Change log heading not nested properly, make it an h2 not h4

For further accessibility details see #1004

**Comment by dolsysmith** [comment]
I'm not seeing any `<i>` tags in `collection_detail.html`. The font-awesome icons there are in `<span>` tags and are already inside `<a>` tags. Was item referring to a different page, perhaps?

**Comment by lwrubel** [comment]
@dsmith, I think that item was a repeat of the collection_set_detail item, and should have only been on collection_set_detail.html.

**Comment by kerchner** [comment]
Based on item 2 in #1023, it would follow that all of the alerts that are not `alert-danger` should have their `role` property changed from `alert` to `status`.  There are several on `collection_detail.html` (as well as other pages - I'll open a separate ticket for those not covered in existing tickets)

**Comment by dolsysmith** [comment]
'm wondering if the headings on the collection_detail page need another look. At the moment, I've made the Change Log an <h2> per the ticket. But it's the only one. The harvest type -- e.g., "Twitter User Timeline" is an <h3>, which doesn't seem quite right. Also, Seeds ad Harvests are <h4>. I think the latter two should be <h2> by analogy with Change Log. But I'm not sure about "Twitter User Timeline," since it's not really a separate section so much as a detail about the Collection name (the  `<h1>`).

**Comment by lwrubel** [comment]
@dolsysmith This echoes what Max commented in the accessibility review google doc. He suggested: 
h1: Name of the collection 
Type of collection (Twitter user timeline) could be a <p> or <span> styled to be larger instead of an <h3>. 
h2: Seeds
h2: Harvests 
h2: Change log

**Comment by lwrubel** [comment]
FWIW, if you think the h2 should be styled a little differently across SFM because of this change, please feel free to do that in this ticket.
